### PR TITLE
Mk public data dumps episode dstb

### PR DESCRIPTION
**Pull request by mkangia**

### CODE REVIEW

File: custom/enikshay/management/commands/data_dumps_person_case.py [hunk 1]
```diff
@@ -19,6 +21,11 @@ class Command(BaseDataDump):
     data dumps for person cases
     https://docs.google.com/spreadsheets/d/1OPp0oFlizDnIyrn7Eiv11vUp8IBmc73hES7qqT-mKKA/edit#gid=1039030624
     """
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.case_type = CASE_TYPE_PERSON
+        self.input_file_name = os.path.join(os.path.dirname(__file__),
+                                            'data_dumps_person_case.csv')
```
Comment (esoergel) [inline custom/enikshay/management/commands/data_dumps_person_case.py]: 👍

File: data_dumps_dstb_episodes.csv [hunk 2]
```diff
@@ -0,0 +1 @@
...
Adverse drug reaction history,self,adr_history
"Date of first reaction/ symptom, if any",N/A,N/A
"Specify reaction, if any",N/A,N/A
"Note on Reactions, if any",N/A,N/A
Any Action taken,N/A,N/A
Date of remedial action,N/A,N/A
"Specify action, if any",N/A,N/A
Duration of management for adverse events (Days),N/A,N/A
Treatment Outcome,self,treatment_outcome
Date of Treatment Outcome,self,treatment_outcome_date
```
Comment (esoergel) [inline data_dumps_dstb_episodes.csv]: Was this file committed by mistake?  It's in the root of the repo, and there's a duplicate in the mgmt cmd dir

**Review by esoergel — CHANGES_REQUESTED** [review]
I think there's an extra file, otherwise looks g2g

File: data_dumps_dstb_episodes.csv (hunk 2 above)
Comment (mkangia) [reply data_dumps_dstb_episodes.csv]: eh? where did that come from.. thanks for catching.. will remove it.

File: custom/enikshay/management/commands/data_dumps_dstb_episodes.csv [hunk 3]
```diff
@@ -1 +1,3 @@
...
Marital Status,N/A,N/A,N/A
Marital Status (Other - Specify),N/A,N/A,N/A
Socio Economic Status,N/A,N/A,N/A
Name & Address of Contact Person,N/A,N/A,N/A
Mobile/ Landline Number of Contact Person,N/A,N/A,N/A
Predominant Symptoms,Primary Case,self,predominant_symptom
"Duration of symptoms in Days, if any",Primary Case,self,symptom_duration
History of ATT greater than one month?,N/A,N/A,N/A
Number of health care providers visited before diagnosis for current episodes,N/A,N/A,N/A
"Months of Treatment undertaken
```
Comment (esoergel) [inline custom/enikshay/management/commands/data_dumps_dstb_episodes.csv]: I notice that the "Type" column is a new addition to these spreadsheets - did you update the base command to account for it?  I guess this uses `csv.DictReader`, so maybe that's not relevant, huh?

**Review by esoergel — APPROVED** [review]

**Pull request merged**